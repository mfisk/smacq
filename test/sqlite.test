#!/bin/sh 
DB=`pwd`/sqlitedb

rm -f $DB

#sqlite3 $DB << EOM
#create table foo(ts varchar(80), srcip varchar(32), dstip varchar(32));
#EOM

${BINDIR}smacqq -m -g <<EOF 
sqlinsert -t foo -d "URI=$DB" ts srcip dstip from (uniq srcip, dstip from pcapfile($TEST/0325@1112-snort.pcap))
EOF

${BINDIR}smacqq -m -g <<EOF 
print -v ts srcip dstip from sqlquery(-d "URI=$DB" foo)
EOF

