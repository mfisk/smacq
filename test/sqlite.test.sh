#!/bin/sh 
DB=`pwd`/sqlitedb

rm -f $DB

#sqlite3 $DB << EOM
#create table foo(ts varchar(80), srcip varchar(32), dstip varchar(32));
#EOM

$SMACQQ -m -g <<EOF 
sqlinsert -t foo -d "URI=$DB" ts srcip dstip from (uniq srcip, dstip from pcapfile(0325@1112-snort.pcap))
EOF

$SMACQQ -m -g <<EOF 
print -v ts srcip dstip from sqlquery(-d "URI=$DB" foo)
EOF

