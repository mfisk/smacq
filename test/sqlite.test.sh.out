ts = 2000-03-25T12:13:29.360668-07:00	srcip = 192.168.0.54	dstip = 192.168.0.210
ts = 2000-03-25T12:15:28.020237-07:00	srcip = 192.168.0.54	dstip = 192.168.0.200
ts = 2000-03-25T12:15:31.922436-07:00	srcip = 192.168.0.200	dstip = 192.168.0.54
